src/VirtoCommerce.SitemapsModule.Data/Extensions/RelativePathUtils.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace VirtoCommerce.SitemapsModule.Data.Extensions;
+
+public static class RelativePathUtils
+{
+    public static string Combine(string parentRelativePath, string childRelativePath)
```

Comment:
I've only found one place where it's used now. There used to be two. Is there any sense of a separate existence now?))

Reply:
Fixed